src/bindings/js/node/lib/addon.ts

```diff
@@ -21,6 +21,8 @@ type elementTypeString =
   | 'f32'
   | 'string';
 
+type OVAny = string | number | boolean;
```

Comment:
Propose to add comment here, how it matches with OV type

Reply:
I am not sure what messages we could leave here for users and not developers. This change is mostly important for future development and maybe the explanation that no matter what user passes here (`string | number | boolean`) it will be processed by the same function to match `ov::Any` should be placed somewhere else.